### PR TITLE
Issue 42 global css style for font

### LINKED ISSUE #42: Global CSS style for font
## Basic Information
Create global CSS style classes for the fonts, see [Figma](https://www.figma.com/design/bhSe5HkMJKKN0Bs1ZqSddi/Penni?node-id=449-90&t=Tx5T1Anqr6fBjZy9-0)

image

e.g. 
```css
.penni-h1 {
    @apply text-3x1 ...
}
```

## Other Information
- **Location**: `/styles/styles.css` (include in `/pages/_app.tsx`)
- **Behaviour**: Make style fonts globally accessible
- **Purpose**: Make the codebase more concise. Instead of defining the same style in different components, we can reuse the same CSS class.

### PR DESCRIPTION
**Pull request by SeiKasahara**
## Change Summary
Created proper font style css in `./client/src/styles/styles.css`.
Applied the Inter font.
Since the SF Pro font cannot be referenced online so waiting for further modification.

## P.S.
Now all the fonts can be tested in `index.tsx` page, if necessary, uncomment the test part and see

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [x] The change has tests
- [ ] The change has documentation

# Related issue

- Resolve #42

### CODE REVIEW

**Comment by yunho7687** [comment]
You'll need to fix the frontend code style issues checks errors before this can be merged.

File: client/src/pages/index.tsx
Comment (yunho7687) [inline client/src/pages/index.tsx:1]: please create another page to show all the use-cases

Comment (yunho7687) [reply client/src/pages/index.tsx:1]: please try running `npm run format` at the client directory

**Comment by SeiKasahara** [comment]
> You'll need to fix the frontend code style issues checks errors before this can be merged.

Fixed all code style to pass the checks

File: client/src/pages/index.tsx [hunk 1]
```diff
@@ -37,7 +37,22 @@ export default function Home() {
         fontSans.variable,
       )}
     >
-      <h1 className="text-3xl text-primary">Test title</h1>
+      <h1 className="title1 text-primary">Test title</h1>
+      {/* Test for font */}
+      {/*
+      <h1 className="largetitle text-primary">Largetitle</h1>
```
Comment (yunho7687) [inline client/src/pages/index.tsx]: please create a new page to show all the use-cases and delete these comments

**Review by yunho7687 — CHANGES_REQUESTED** [review]

File: client/src/pages/index.tsx (hunk 1 above)
Comment (SeiKasahara) [reply client/src/pages/index.tsx]: Completed modification in creating `font` page in `font.tsx`
To access it plz use `localhost:3000/font`

**Review by yunho7687 — APPROVED** [review]

**Pull request merged**